### PR TITLE
EN-55620: Increase timeout for scanShape calls in the GIS job type

### PR DESCRIPTION
**Pull request by rjmac**
Also correct a bug whereby I think that calls with a proxy set would have no timeout?

### CODE REVIEW

**Review by saraid — APPROVED** [review]
Makes sense.

File: src/main/java/com/socrata/datasync/publishers/GISPublisher.java [hunk 1]
```diff
@@ -205,15 +206,16 @@ private static String[] getStatus(URI uri,
...
+                                         int timeoutMS) throws IOException {
         HttpUtility httpUtility = new HttpUtility(userPrefs, true, 3, 2);
 
         System.out.println("Posting file...");
         HttpEntity httpEntity = MultipartEntityBuilder.create()
             .addBinaryBody(file.getName(), file, ContentType.APPLICATION_OCTET_STREAM,file.getName())
             .build();
 
-        HttpResponse response = httpUtility.post(uri, httpEntity);
+        HttpResponse response = httpUtility.post(uri, httpEntity, timeoutMS);
```
Comment (rlvoyer) [inline src/main/java/com/socrata/datasync/publishers/GISPublisher.java:218]: What was the default timeout before this change?

**Review by rlvoyer — APPROVED** [review]
Looks good to me!

File: src/main/java/com/socrata/datasync/publishers/GISPublisher.java (hunk 1 above)
Comment (rjmac) [reply src/main/java/com/socrata/datasync/publishers/GISPublisher.java:218]: 1 minute (it was using the default timeouts set above)

**Pull request merged**